src/config/fonts/styles.css

```diff
@@ -1,96 +1,55 @@
 @font-face {
-  font-family: 'Arizona Serif';
-  src: url('./ABCArizonaSerif-Light.ttf') format('truetype');
-  font-style: normal;
-}
-
-@font-face {
-  font-family: 'Arizona Serif';
-  src: url('./ArizonaSerif-Light.woff') format('woff');
+  font-family: 'ArizonaSerif';
+  src: url('./ArizonaSerif/ArizonaSerif-Light.otf') format('truetype');
```

Comment:
I think this has to be very specific, `format(...)` has to match the file format:
- `*.otf` has to be `format(opentype)`
- `*.ttf` has to be `format(truetype)`
- `*.woff` has to be `format(woff)`
- `*.woff2` has to be `format(woff2)`
Check the rest of the file to match them correctly,